docs/local_subscriber.md

```diff
@@ -0,0 +1,24 @@
+LocalSubscriber
+===============
+The LocalSubscriber block subscribes to the configured topic and outputs signals received. Only LocalPublisher blocks on the same nio instance can send data to the LocalSubscriber blocks. Unlike the Subscriber block, these signals do not need to contain data that is valid JSON.
+
+Properties
+----------
+- **local_identifier**: Hidden property with a default of `[[INSTANCE_ID]]`. Unique identifier of this instance in the nio system.
+- **topic**: Hierarchical topic string to subscribe to.
```

Comment:
Let's also make sure that the properties display their Title here, which in this case just means it's capitalized.

Do we have any explanation of "hierarchical topic string"?